.github/workflows/vcpkg.yml

```diff
@@ -90,6 +90,12 @@ jobs:
         run: |
           VCPKG_BASH_PATH=${VCPKG_INSTALLATION_ROOT//\\//}
           echo "VCPKG_INSTALLATION_ROOT=$VCPKG_BASH_PATH" >> "$GITHUB_ENV"
+      
+      - name: Update vcpkg
+        shell: bash
+        run: |
+          git -C ${{ env.VCPKG_INSTALLATION_ROOT }} clean -fdx
+          git -C ${{ env.VCPKG_INSTALLATION_ROOT }} pull
```

Comment:
This mean to be on the latest vcpkg. I don't recommended it. Because many time the master is got broken.

Reply:
Why not wait for every two weeks update? why you want it up to date?